src/pages/Groups/Group3/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+`
+
+export const HeaderRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 10px 0px;
+`
+
+export const HeaderInformation = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+export const HeaderBackButton = styled.img`
+  height: 20px;
+  width: 20px;
+  margin: 0px 10px 0px 0px;
+`
+
+export const HeaderSearchButton = styled.img`
+  height: 20px;
+  width: 20px;
+  margin: 0px 10px;
+`
+
+export const Header = styled.h1`
+  font-size: 32px;
+  margin: 0;
+`
+
+export const StyledSegment = styled.div``
```

Comment:
if they do not have any styling, use `<div></div>` instead

Reply:
resolved in commit b63baef